### PR TITLE
Moved tests out of run_in_graph_and_eager_mode in conditional gradient

### PR DESCRIPTION
**Pull request by autoih**
cc @gabrieldemarmiesse

### CODE REVIEW

File: tensorflow_addons/optimizers/tests/conditional_gradient_test.py [hunk 1]
```diff
@@ -571,409 +571,377 @@ def test_tensor_learning_rate_and_conditional_gradient_frobenius(dtype):
...
+    num_samples = len(db_grad)
+    var0 = tf.Variable([0.0] * num_samples)
+    grads0 = tf.constant([0.0] * num_samples)
+    ord = "fro"
+    cg_opt = cg_lib.ConditionalGradient(learning_rate=0.1, lambda_=0.1, ord=ord)
+
+    for i in range(num_samples):
+        grads0 = tf.constant(db_grad[i])
+        cg_opt.apply_gradients(zip([grads0], [var0]))
+        np.allclose(np.array(db_out[i]), var0.numpy())
```
Comment (gabrieldemarmiesse) [inline tensorflow_addons/optimizers/tests/conditional_gradient_test.py:847]:
```suggestion
        np.testing.assert_allclose(np.array(db_out[i]), var0.numpy())
```

np.allclose returns a boolean.

**Comment by gabrieldemarmiesse** [comment]
Hooo boy, this no space left on device...

**Comment by gabrieldemarmiesse** [comment]
Everything is passing except the linux build with the no space left on device issue. Let's merge it anyway.
